app/helpers/application_helper.rb

```diff
@@ -60,4 +60,10 @@ def cancel_link_text
   def desktop_device?
     !BrowserCache.parse(request.user_agent).mobile?
   end
+
+  def selfie_status_from_response(client_response)
+    return client_response.selfie_status if client_response.respond_to?(:selfie_status)
```

Comment:
Is the `respond_to?` check how we are managing the 50-50 state?

If this code is running, it will be on a new box, so in that context, the `selfie_status` property will exist no matter what. I think we need to check the return values of the properties (because we really want to check the keys of the serialized JSON but that's hidden at this layer) so all we can see is which values are `nil`